wicket-util/src/main/java/org/apache/wicket/util/value/HeaderItemAttributeMap.java

```diff
@@ -0,0 +1,24 @@
+package org.apache.wicket.util.value;
+
+import java.util.Map;
+
+public class HeaderItemAttributeMap extends AttributeMap {

```

Comment:
I don't see much value in this class.

Reply:
I mainly need it to have put and add below accepting the enum. Is there any other mechanism to avoid doing `attributes.put(SCRIPT_SRC.getName(), val)` every time? I mean the .getName()